pyblish_qml/control.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, host, parent=None, targets=[]):
             },
             "comment": "",
             "firstRun": True,
-            "onPublished": False,
+            "testPassed": False,
```

Comment:
Ah I think I see. It's application state? Let's put that in `self.data["state"]` with the other application state variables. Looks like `firstRun` should have been in there too, I can't remember when that got added or by whom. It's not the end of the world, but let's not add to the inconsistency.

Reply:
> It's application state?

Ah, Yes !

And okay, moving it :)
-> 64c3db